lib/dateTime.js

```diff
@@ -1128,10 +1128,24 @@ export function formatDateTime(date, options) {
 
 }
 
-export function formatDateTimeFromTimestamp(timestamp, options) {
+function parseLocalDateTimeFromTimestamp(timestamp, options) {
```

Comment:
Methods which returned JS `Date` objects seemed to be named `parse<dateFormat>` so I tried to keep that consistent here.